ui/components/src/TimeSeriesTooltip/TimeSeriesTooltip.tsx

```diff
@@ -113,31 +108,17 @@ export const TimeSeriesTooltip = React.memo(function TimeSeriesTooltip({
           transform: cursorTransform,
         }}
       >
-        <TooltipContent
-          series={focusedSeries}
-          wrapLabels={wrapLabels}
-          isTooltipPinned={isTooltipPinned}
-          onUnpinClick={() => {
-            setPinnedPos(null);
-            if (onUnpinClick !== undefined) {
-              onUnpinClick();
-            }
-          }}
-        />
-        {showAllSeriesToggle && (
-          <Stack direction="row" gap={1} alignItems="center" sx={{ textAlign: 'right' }}>
-            <Typography>Show All?</Typography>
-            <Switch
-              checked={showAllSeries}
-              onChange={(_, checked) => setShowAllSeries(checked)}
-              sx={(theme) => ({
-                '& .MuiSwitch-switchBase': {
-                  color: theme.palette.common.white,
-                },
-              })}
-            />
-          </Stack>
-        )}
+        <Stack spacing={0.5}>
+          <TooltipHeader
+            seriesTimeMs={seriesTimeMs}
+            totalSeries={totalSeries}
```

Comment:
more up to you stuff: Since `seriesTimeMs` and `totalSeries` are mainly used by the `TooltipHeader` (as far as I can see), just pass in `series` to `TooltipHeader` and have the component derive these values from `series`.

Reply:
Ended up moving seriesTimeMs into TooltipHeader but kept totalSeries in TimeSeriesTooltip (it'll be needed when I go to virtualize the tooltip content in the future)